### PR TITLE
Bootstrap salt with -P

### PR DESCRIPTION
**Pull request by jpic**
Thanks @GregMSThompson for the heads up:

https://github.com/saltstack-formulas/jenkins-formula/pull/24#issuecomment-126624677

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ env:
 
 before_install:
   - sudo apt-get update
-  - curl -L http://bootstrap.saltstack.org | sudo -E sh -s -- stable
+  - curl -L http://bootstrap.saltstack.org | sudo -P -E sh -s -- stable
```
Comment (iggy) [inline .travis.yml]: itym (move -P after sh)
```
  - curl -L http://bootstrap.saltstack.org | sudo -E sh -s -P -- stable
```

**Comment by jpic** [comment]
Apparently I'm doing it wrong ...

```
$ curl -L http://bootstrap.saltstack.org | sudo -E sh -s -P -- stable

sh: 0: Illegal option -P

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current

                                 Dload  Upload   Total   Spent    Left  Speed

100   256  100   256    0     0   2321      0 --:--:-- --:--:-- --:--:--  5953

  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0

The command "curl -L http://bootstrap.saltstack.org | sudo -E sh -s -P -- stable" failed and exited with 2 during .
```

**Comment by GregMSThompson** [comment]
Hi jpic, try moving the -P to after the -- so it's applied into the shell session i.e. curl -L http://bootstrap.saltstack.org | sudo -E sh -s -- -P stable. Hopefully, that'll fix it :).

**Comment by jpic** [comment]
Thx, I need to find some time to concentrate on this xD

**Comment by jpic** [comment]
We shouldn't have merged this, it doesn't pass tests and it's loaded with fixup commits which should be squashed.

**Comment by nmadhok** [comment]
@jpic Reverting this.

**Pull request merged**